package.json

```diff
@@ -77,6 +77,15 @@
     "delay": 2500,
     "ext": "js,json,html,njk"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
```

Comment:
In #532 we upgraded to Husky 7, which changes the way hooks are defined. They're now defined as scripts like `pre-commit` in the repo's `.husky` directory. So I reckon that you'll need to take these out of here and put them in` .husky` (perhaps you can move the contents of the `hooks` directory into a directory called e.g. `.husky/git-lfs`, too?)